iaso/api/mobile/entity.py

```diff
@@ -186,7 +186,7 @@ def get_queryset(self):
         if not app_id:
             raise ParseError("app_id is required")
 
-        queryset = get_queryset_for_user_and_app_id(user, app_id).filter(deleted_at__isnull=True)
+        queryset = filter_on_user_and_app_id(Entity.objects, user, app_id)
```

Comment:
Why remove `.filter(deleted_at__isnull=True)`?

Reply:
Entity is a `SoftDeletableModel` so the filter is there by default, this extra one would just add a second `"iaso_entity"."deleted_at" IS NULL` to the generated SQL query.